### PR TITLE
Avoid warning message from ssh-keygen

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4296/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -2380,12 +2380,14 @@ def erase_old_data
...
-        Djinn.log_run("ssh-keygen -R #{node.private_ip}")
-        Djinn.log_run("ssh-keygen -R #{node.public_ip}")
-      }
+    if File.exist?(File.expand_path("~/.ssh/known_hosts"))
+      @state_change_lock.synchronize {
+          @nodes.each { |node|
+          Djinn.log_run("ssh-keygen -R #{node.private_ip}")
+          Djinn.log_run("ssh-keygen -R #{node.public_ip}")
+        }
     }
```
Comment (cdonati) [inline AppController/djinn.rb]: Please indent this bracket for clarity.

File: AppController/djinn.rb [hunk 2]
```diff
@@ -2380,12 +2380,14 @@ def erase_old_data
     Djinn.log_run("rm -f ~/.appscale_cookies")
 
     # Delete (possibly old) mapping of IP <-> HostKey.
-    @state_change_lock.synchronize {
-      @nodes.each { |node|
-        Djinn.log_run("ssh-keygen -R #{node.private_ip}")
-        Djinn.log_run("ssh-keygen -R #{node.public_ip}")
-      }
+    if File.exist?(File.expand_path("~/.ssh/known_hosts"))
```
Comment (cdonati) [inline AppController/djinn.rb]: Rubocop would be displeased with you using double quotes here.

File: scripts/fast-start.sh [hunk 3]
```diff
@@ -10,6 +10,11 @@
 # ssh -i <key> ubuntu@<public IP> 'curl -Lo - fast-start.appscale.com|sudo -i sh'
 #
 
+# On some system, when running this scipt from rc.local (ie at boot time)
```
Comment (cdonati) [inline scripts/fast-start.sh]: systems

**Pull request merged**